test/specs/rules/require-template-strings.js

```diff
@@ -42,6 +42,20 @@ describe('rules/require-template-strings', function() {
         }
     );
 
+    it('should report for the use of string concatenation with a CallExpression',
+        function() {
+            assert(checker.checkString('a() + "a"').getErrorCount() === 1);
+            assert(checker.checkString('"a" + a()').getErrorCount() === 1);
+        }
+    );
+
+    it('should report for the use of string concatenation with a MemberExpression',
+        function() {
+            assert(checker.checkString('a.b + "a"').getErrorCount() === 1);
```

Comment:
What about for tagged template strings being concatenated with another string or template string. I guess I'm questioning the ast type for a tagged template string.